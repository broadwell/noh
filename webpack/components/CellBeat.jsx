import React from "react";
import PropTypes from "prop-types";

const CellBeat = props => (
  <div className="cell cell--1 cell--beat">{props.beatText}</div>
);

CellBeat.propTypes = {
  beatText: PropTypes.string.isRequired
};

export default CellBeat;
