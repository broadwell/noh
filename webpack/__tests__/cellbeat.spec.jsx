import React from "react";
import { shallow } from "enzyme";
import CellBeat from "../components/CellBeat";

describe("<CellBeat>", () => {
  it("renders as expected by default", () => {
    const component = shallow(<CellBeat beatText="1" />);
    expect(component).toMatchSnapshot();
  });
});
