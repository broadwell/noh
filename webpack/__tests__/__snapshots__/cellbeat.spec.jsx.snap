// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CellBeat> renders as expected by default 1`] = `
<div
  className="cell cell--1 cell--beat"
>
  1
</div>
`;
